src/SFA.DAS.Reservations.Web/Infrastructure/EmployerAccountAuthorizationHandler.cs

```diff
@@ -76,5 +87,15 @@ public bool IsEmployerAuthorised(AuthorizationHandlerContext context)
             
             return true;
         }
+
+        private static bool CheckUserRoleForAccess(EmployerIdentifier employerIdentifier)
+        {
+            if (!Enum.TryParse<EmployerUserRole>(employerIdentifier.Role, true, out var userRole))
+            {
+                return false;
+            }
+
+            return userRole != EmployerUserRole.Viewer && userRole != EmployerUserRole.None;
```

Comment:
I think id rather this was 
```suggestion
            return userRole == EmployerUserRole.Owner || userRole == EmployerUserRole.Transactor;
```
otherwise if a new role was added like reporter, then we would automatically grant them access on the current implementation